live-build/config/hooks/vm-artifacts/90-linux-migration-artifact.binary

```diff
@@ -76,6 +76,10 @@ mkdir $DEPOT_DIRECTORY
 #
 (
 	cd binary
+
+	# Copy an illumos version of dx_manage_pg for migration purposes.
+	cp ../migration-scripts/dx_manage_pg_illumos opt/delphix/server/bin/
```

Comment:
Will this script only be used for migration? If so, why do we need it installed in the root filesystem, rather than the migration process using the script directly from the upgrade directory (like the "dx" scripts)?

If we _need_ the file to be installed in the root directory like this, I agree with John, that we should install the file via some package (i.e. the app-gate). If we don't need it to be installed, I'd be OK leaving it in the "migration-scripts" directory of this repository, and then having consumers execute it directly from the unpacked directory in "/var/dlpx-update".

Reply:
It might be best to just keep current `dx_manage_pg` that works for both linux and illumos, due to the requirements of migration. I feel like removing the illumos-specific code from the app-gate script, only to have it duplicated here, makes things more complicated for little benefit.